Added FAQs (#7708)
chef/automate: components/docs-chef-io/content/automate/centralizing_log.md

```

The above step will let you view the `automate.log` file in your splunk forwarder. Click [here](https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html) to know about the Splunk Forwarder.

## Frequently Asked Questions

> ### Is the setting of `redirect_log_file_path` directory where the log files will be stored or the full path of the actual log file?

Reviewer: This does not look like a FAQ to me. This can be resolved by being more explicit in our document

- `redirect_log_file_path` is a directory where the automate.log file will be created.

### Patching the log config. Should we run one command to patch the whole cluster from the bastion host?

> - Run multiple commands, e.g.:

Reviewer: We should include all the steps with more details. 
- Mount an NFS or network-attached storage to all the nodes. Choose a log location
- command to patch Automate nodes with the same log location
- command to patch Server nodes with the same log location
- command to patch PG nodes with the same log location
- command to patch OS nodes with the same log location

```

The above step will let you view the `automate.log` file in your splunk forwarder. Click [here](https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html) to know about the Splunk Forwarder.

## Log File

> Create the centralizing log file, i.e., **automate.log** in the `redirect_log_file_path` directory.

Author: @dishanktiwari2501 Please change the line as 
```suggestion
NOTE: `redirect_log_file_path`  is a directory where automate.log gets created.
```

@kalroy  what do you say?
